File: Day_11/Q41.java
import java.util.*;
class Q41{
    //Calculating Sum Of Two Numbers
        static int sum(int a,int b){
        return (a+b);
    }
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        int a = sc.nextInt();
        int b = sc.nextInt();
        //Printing The Result
        System.out.print(sum(a,b));
        sc.close();
    }
}

File: Day_11/Q41.cpp
#include <iostream>
using namespace std;
//Calculating Sum Of Two Numbers
int sum(int a,int b){
    return (a+b);
}
int main(){
    int a,b;
    cin >> a >> b;
    //Printing The Result
    cout << sum(a,b);
    return 0;
}